assemble/conf/accumulo-env.sh

```diff
@@ -89,14 +89,22 @@ JAVA_OPTS=(
 ## JVM options set for individual applications
 # cmd is set by calling script that sources this env file
 #shellcheck disable=SC2154
-case "$cmd" in
-  manager) JAVA_OPTS=('-Xmx512m' '-Xms512m' "${JAVA_OPTS[@]}") ;;
-  monitor) JAVA_OPTS=('-Xmx256m' '-Xms256m' "${JAVA_OPTS[@]}") ;;
-  gc) JAVA_OPTS=('-Xmx256m' '-Xms256m' "${JAVA_OPTS[@]}") ;;
-  tserver) JAVA_OPTS=('-Xmx768m' '-Xms768m' "${JAVA_OPTS[@]}") ;;
-  compactor) JAVA_OPTS=('-Xmx256m' '-Xms256m' "${JAVA_OPTS[@]}") ;;
-  sserver) JAVA_OPTS=('-Xmx512m' '-Xms512m' "${JAVA_OPTS[@]}") ;;
-  *) JAVA_OPTS=('-Xmx256m' '-Xms64m' "${JAVA_OPTS[@]}") ;;
+group="${ACCUMULO_RESOURCE_GROUP:-default}"
+case "$group" in
```

Comment:
I think you could avoid creating the intermediate variable, since it could be confusing to call it group in one script, convert it to ACCUMULO_RESOURCE_GROUP, then convert it to the opposite here.

I think you can just do:

```suggestion
case "${ACCUMULO_RESOURCE_GROUP:-default}" in
```

Reply:
Suggestion applied in 0bc61ef